### PR TITLE
Create SensorData interface and MockSensorData

### LINKED ISSUE #14: Add MockSensorData
As noted in #13, we're currently returning `null` from one of our mocks because `SensorData` has no mock object. That situation should be improved.

### PR DESCRIPTION
**Pull request by a2aaron**
I've also added SensorData to the MockRobot. Let me know if I've done something wrong or if there's more to be added.

This resolves #14.

### CODE REVIEW

**Comment by porglezomp** [comment]
Aaron, it seems you're on a badly named branch. It's not super important, but keep it in mind for the future.

**Comment by porglezomp** [comment]
If you're able to rename branches on GitHub, then I recommend doing that.

File: FRC1076-2016-Competition/test/org/usfirst/frc/team1076/test/mock/MockRobot.java [hunk 1]
```diff
@@ -8,7 +8,7 @@
 	public double left, right, arm, intake;
 	public double lidarSpeed;
 	public boolean brakes;
-	// TODO: Add SensorData
+	public SensorData sensorData;
```
Comment (porglezomp) [inline FRC1076-2016-Competition/test/org/usfirst/frc/team1076/test/mock/MockRobot.java:11]: This should be an `ISensorData`, not a `SensorData`.

File: FRC1076-2016-Competition/test/org/usfirst/frc/team1076/test/mock/MockRobot.java [hunk 2]
```diff
@@ -41,8 +41,7 @@ public void setLidarSpeed(double speed) {
 	}
 
 	public SensorData getSensorData() {
```
Comment (porglezomp) [inline FRC1076-2016-Competition/test/org/usfirst/frc/team1076/test/mock/MockRobot.java:43]: The interface should be changed to return an `ISensorData`, and all overriding methods should be `@Override`.

**Comment by porglezomp** [comment]
Try to change everywhere that we're using `SensorData` to `ISensorData`.

**Comment by a2aaron** [comment]
Closed. See PR #21

**Pull request closed without merging**